WVAanalysisRun2/src/setOutputTree.cc

```diff
@@ -100,6 +101,8 @@ void setOutputTree::setBranches()
   fTree->Branch("isBoostedSignal",&isBoostedSignal,"isBoostedSignal/I");
   fTree->Branch("wSampleWeight",&wSampleWeight,"wSampleWeight/F");
   fTree->Branch("genWeight",&genWeight,"genWeight/F");
+  fTree->Branch("originalWeight",&originalWeight,"originalWeight/D");
```

Comment:
This only needs to be modified to say:
fTree->Branch("originalWeight",&originalWeight,"originalWeight/F");